de.metas.issue.tracking.github/src/main/java/de/metas/issue/tracking/github/api/v3/service/GithubClient.java

```diff
@@ -86,7 +86,9 @@ public GithubClient(final RestService restService)
 
 		issueList = ImmutableList.copyOf(restService.performGet(getRequest, Issue[].class).getBody());
 
-		return issueList;
+		return issueList.stream()
+				.filter(issue -> issue.getPullRequest() == null)
```

Comment:
what are we doing here?

Reply:
filterring out PRs as GitHub will send those too via the `issue` endpoints.

Reply:
allright, pls make it clearer in the code. E.g. add this as comment to the code, or (better) e.g. add issue.isPullRequest and filter by that